src/content/content.npmjs.js

```diff
@@ -13,10 +15,28 @@ const addPackageReport = (packageID) => {
   }
 };
 
-mountContentScript(async () => {
+browser.runtime.onMessage.addListener(function ({ message, url }) {
+  if (message === EVENT_URL_CHANGE) {
+    const packageId = new URL(url).pathname.replace('/package/', '');
+    console.log(`package changed to ${packageId}, reloaing package info`);
+    reloadPackageInfo();
+  }
+});
```

Comment:
I wanted all the events to be defined in one place, to get control over all the events sended from here to there.

First, it's means you should define this event listener on the `content-events.js` file, and export it to use it here.

Second, I'm thinking what if other websites needs the same logic? Maybe we can implement a flow like the `mountContentScript` function?
For example, a function called `reloadWhenURLChanged` that accept the removing existing element and the `loadPackageInfo` function, and listening to the `EVENT_URL_CHANGE` to run those functions.

Reply:
> Do you think you can add a Playwright test for this use case?
> 
> (I'm warning you, I failed to add a test for npmjs.com, don't know why wink. #111)

I never used Playwright before but I'll give it a try.